ci/coverity.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+  - job: Coverity
+    pool:
+      name: MLNX
+      demands: ucx_docker
+    container: centos7
+    displayName: Coverity
+
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 10
+
+      - bash: |
+          set -Exeuo pipefail
+          source /etc/profile.d/modules.sh
+          module load /hpc/local/etc/modulefiles/tools/cov-2021.12
+          ./autogen.sh
+          kernel_ver=$(rpm -qa | grep kernel-devel | cut -d'-' -f3-)
+          ./configure --with-kerneldir=/usr/src/kernels/${kernel_ver}
+          cov-build --dir cov-dir make -j$(nproc) &>/dev/null
+          cov-manage-emit --dir cov-dir --tu-pattern "file('.*/kernel/*')" delete || true
```

Comment:
seems we are also excluding `xpmem/kernel`. maybe use regexp `/usr/.*/kernel` or first -tu include `xpmem/kernel`?

Reply:
Proper error reporting based on Elena's suggestion:
https://dev.azure.com/ucfconsort/xpmem/_build/results?buildId=70830&view=logs&j=03fdef1b-ad3f-5d26-daac-16fc56fa63ed&t=41e24686-70b1-5db5-0fd6-6e2a93ac9df9

Reply:
merged #50, do we want to make coverity failure mandatory?

Reply:
ok let's make it mandatory

Reply:
Done